internal/clients/debug.go

```diff
@@ -38,7 +38,7 @@ func newDebugTransport(name string, transport esapi.Transport) *debugTransport {
 
 func (d *debugTransport) Perform(r *http.Request) (*http.Response, error) {
 	ctx := r.Context()
-	reqData, err := httputil.DumpRequestOut(r, true)
+	reqData, err := httputil.DumpRequest(r, true)
```

Comment:
From the [docs](https://pkg.go.dev/net/http/httputil#DumpRequest)

> It should only be used by servers to debug client requests

I think that we should be either using the build in go-elasticsearch [loggers](https://github.com/elastic/go-elasticsearch/blob/main/_examples/logging/default.go) or if using `tflog.*` was important implementing a [custom logger](https://github.com/elastic/go-elasticsearch/blob/main/_examples/logging/custom.go)